### PR TITLE
Allowed for specifying prove layout to the docker image

### PR DESCRIPTION
**Pull request by matzayonc**

### CODE REVIEW

File: cairo-prover-entrypoint.sh [hunk 1]
```diff
@@ -3,19 +3,20 @@
...
     --memory_file program_memory.memory \
-    --layout recursive \
+    --layout $layout \
     --proof_mode \
     --air_public_input program_public_input.json \
     --air_private_input program_private_input.json \
     --args_file program_input.txt \
     program.sierra.json \
-    2>&1 > /dev/null && \
+    2>&1 > /dev/null
```
Comment (neotheprogramist) [inline cairo-prover-entrypoint.sh]: leave `&& \` not to execute next line if failed

File: example/cairo/src/lib.cairo [hunk 2]
```diff
@@ -1,7 +1,7 @@
 mod utils;
 
-fn main(n: felt252) -> Array<felt252> {
-    array![utils::factorial(n)]
+fn main(n: felt252, c: felt252) -> Array<felt252> {
```
Comment (neotheprogramist) [inline example/cairo/src/lib.cairo:3]: why `c`?

**Review by neotheprogramist — CHANGES_REQUESTED** [review]

File: example/cairo/src/lib.cairo (hunk 2 above)
Comment (matzayonc) [reply example/cairo/src/lib.cairo:3]: just to test with multiple arguments

Comment (matzayonc) [reply example/cairo/src/lib.cairo:3]: and the name `c` is from analogy to the integral

Comment (neotheprogramist) [reply example/cairo/src/lib.cairo:3]: g!

File: cairo-prover-entrypoint.sh (hunk 1 above)
Comment (matzayonc) [reply cairo-prover-entrypoint.sh]: done

**Review by neotheprogramist — APPROVED** [review]
LGTM

**Pull request merged**